src/api/parser/README.md

```diff
@@ -10,8 +10,9 @@ pnpm install
 
 ## Usage
 
-# dev mode with automatic restarts
+### dev mode with automatic restarts
 
+```
 pnpm start
```

Comment:
```suggestion
pnpm dev
```

Reply:
@TueeNguyen an you make this fix please?